getting-started.adoc

```diff
@@ -87,6 +87,24 @@ RHMAP provides a Raincatcher project template to help you run the demonstration.
 
 .. Click *Deploy Cloud App*.
 
+. Upgrade the database into *WFM Demo Project* and *WFM Auth Service*:
```

Comment:
suggestion:
Upgrade the database for the *Cloud App* and the *WFM Auth Service*: